src/renderer/components/uielements/slider/Slider.tsx

```diff
@@ -15,11 +15,21 @@ const Slider: React.FC<Props> = ({
   className = '',
   tooltipPlacement = 'bottom',
   withLabel = false,
+  tipFormatter = (value) => `${value}%`,
   ...rest
 }): JSX.Element => {
+  const ref = useRef()
+  const getTooltipPopupContainer = useCallback((container: HTMLElement) => ref.current || container, [ref])
   return (
     <>
-      <SliderWrapper className={`slider-wrapper ${className}`} tooltipPlacement={tooltipPlacement} {...rest} />
+      <SliderWrapper
+        ref={ref}
+        className={`slider-wrapper ${className}`}
+        tooltipPlacement={tooltipPlacement}
+        getTooltipPopupContainer={getTooltipPopupContainer}
+        tipFormatter={tipFormatter}
+        {...rest}
```

Comment:
[suggestion] Avoid to use `...rest`. _I know we have such things in BEPSwap and just migrated these things into ASGARDEX before - but it's just unsafe..._

Reply:
same here
```ts
import { SliderSingleProps } from 'antd/lib/slider'
interface Props extends SliderSingleProps {
  className?: string
  tooltipPlacement?: TooltipPlacement
  withLabel?: boolean
}
```

Reply:
Yep, that's an artifact of BEPSwap as well. Check out `SliderWrapper` and you will see there is no need to pass any other props. And if there is a need, anyone can create a custom styled component from `Slider`. 

IMO using spread operators for props and pushing "unknown" props from it into nested components is not a good idea. Imagine the case for "hidden" props (hidden in a sense of reading the code), which changes the behavior of the nested components you don't want.